airflow/exceptions.py

```diff
@@ -150,10 +150,6 @@ class DuplicateTaskIdFound(AirflowException):
     """Raise when a Task with duplicate task_id is defined in the same DAG"""
 
 
-class SerializedDagNotFound(DagNotFound):
```

Comment:
Another option would be to use `try_except` in all the places we called `get_dag`